src/main/java/duke/Ui.java

```diff
@@ -9,8 +9,8 @@
  */
 public class Ui {
     private static final String separators = "____________________________________________________________";
```

Comment:
Refer back to the Coding Standard for static final constant, all letters should be capitalized.